templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig

```diff
@@ -450,10 +453,10 @@
 
                                     <dp-date-range-picker
                                         class="u-2-of-4"
-                                        :data-cy="{
+                                        :data-cy="JSON.stringify({
```

Comment:
shouldn't this be something like `institutionPeriod` ?
The comeonent then appends `:endDate` and `startDate` 
https://github.com/demos-europe/demosplan-ui/blob/0c690b83c39d396b24c8ff2320537605e72a99b0/src/components/DpDateRangePicker/DpDateRangePicker.vue#L14

Reply:
Good to know, thanks for spotting